### DISCUSSION TITLE
override prefix

### DISCUSSION THREAD

**Discussion by fortmarek** [Shopify/flash-list#272] [General]
For `overrideItemLayout` and `overrideItemType`, the `override` prefix seems unnecessary - we provide a default if user does not specify the prop, otherwise we use what's provided by the user. I don't see a reason for the `override` prop to be present.

What do you think about removing the prefix? @naqvitalha @davebcn87 @ElviraBurchik 

The discussion originated from [here](https://github.com/Shopify/flash-list/pull/266#discussion_r839419001)

**Comment by ElviraBurchik** [comment]
I don't have a strong opinion here, but to me `override` prefix sounds logical: we have a default value under the hood, but if user provides it - we override it. 

However, I agree that from another point of view it might sound unnecessary. I'm okay with removing the prefix too :raised_hands:

**Reply by fortmarek** [reply]
> to me override prefix sounds logical: we have a default value under the hood, but if user provides it - we override it.

then every prop that has a default should have an override prefix - which imho leads to only hard-to-read code without providing too much value. And `FlatList` props also have defaults without `override` prefix, so _I_ see no reason why to add it for ours 🚫

**Comment by davebcn87** [comment]
As I said in the original conversation I find it unnecessary as well.
